source/renderer/app/api/api.ts

```diff
@@ -460,18 +460,19 @@ export default class AdaApi {
       parameters: request,
     });
     const { walletId, order, fromDate, toDate, isLegacy } = request;
+
     const params = Object.assign(
       {},
       {
         order: order || 'descending',
-        start: undefined,
-        end: undefined,
+        ...(fromDate
```

Comment:
Quick question: Was that problem caused by `start` and `end` properties having `undefined` as their values while they should not even exist on the `params` object?
Maybe it would be good also to secure the place where those properties are used by checking if their values represent correct date.

Reply:
The problem came from the API. It expects the value to be "ISO 8601 date-and-time format".